analyzer/src/main/kotlin/managers/GoMod.kt

```diff
@@ -288,13 +288,26 @@ class GoMod(
 
         // See https://pkg.go.dev/text/template for the format syntax.
         val list = run(
-            "list", "-deps", "-f", "{{with .Module}}{{.Path}} {{.Version}}{{end}}", "-buildvcs=false", "./...",
+            "list", "-deps", "-json=Module", "-buildvcs=false", "./...",
             workingDir = projectDir
         )
 
-        list.stdout.lines().forEach { line ->
-            val columns = line.splitOnWhitespace()
-            if (columns.size in 1..2) result += columns.first()
+        data class DepInfo(
+            @JsonProperty("Module")
+            val module: ModuleInfo? = null
+        )
+
+        list.stdout.byteInputStream().use { inputStream ->
+            JsonFactory().createParser(inputStream).apply {
+                codec = ObjectMapper()
+                nextToken()
+
+                while (hasCurrentToken()) {
+                    val moduleInfo = jsonMapper.readValue(this, DepInfo::class.java).module
+                    if (moduleInfo != null) result += moduleInfo.path
+                    nextToken()
+                }
+            }
```

Comment:
This can be simplified to

    jsonMapper.createParser(list.stdout).use { parser ->
        val moduleIterator = jsonMapper.readValues<DepInfo>(parser)
        moduleIterator.asSequence().mapNotNullTo(result) {
            it.module?.path
        }
    }

BTW, there's similar existing code starting at line 232 that could be simplified in a similar way. If you'd don't feel like simplifying that code in a new commit in this PR, too, I can do it in a follow-up PR.